insertion_sort.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
+
+def insertion_sort(items, n, direction=0):
+    """
+    Given a list of items, insertion_sort will sort
+    the list up to n members. direction of 0 sort the
+    list in an ascending manner. Other numbers sort
+    in descending order.
+    """
+    for i in range( 1, n ):
+        j = i
+        if not bool( direction ):
```

Comment:
`if not direction:` would work, you won't have to convert to bool as 0 is considered False